### ISSUE TITLE
SmartCity-Accessibility Server Side Unit Tests #717 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#383]
Build 'SmartCity-Accessibility Server Side Unit Tests' is failing!

Last 50 lines of build :
```
[...truncated 23.91 KB...]
lat is : 31.901406 lng is : 35.200473
lat is : 31.777545 lng is : 35.221896
lat is : 32.060276 lng is : 34.764359
lat is : 32.078398 lng is : 34.78706
lat is : 32.061957 lng is : 34.783662
lat is : 31.784352 lng is : 35.117952
lat is : 31.778306 lng is : 35.216163
lat is : 32.124488 lng is : 34.825184
lat is : 32.108255 lng is : 34.838057
lat is : 32.069621 lng is : 34.793664
lat is : 32.077567 lng is : 34.789139
lat is : 32.114059 lng is : 34.824801
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.207 sec - in smartcity.accessibility.search.NearbyPlacesTest
Running smartcity.accessibility.search.SearchTest
arrived to search on complete
Tests run: 3, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 0.323 sec - in smartcity.accessibility.search.SearchTest

Results :

Tests in error: 
  AdressSearchServiceTest.searchSuccess » IllegalState Failed to load Applicatio...
  LocationsInRadiusServiceTest.locationtest1 » IllegalState Failed to load Appli...
  LoginServiceTest.UserInfoSuccess » IllegalState Failed to load ApplicationCont...
  LoginServiceTest.loginSuccess » IllegalState Failed to load ApplicationContext
  LoginServiceTest.loginUserDoesNotExist » IllegalState Failed to load Applicati...
  LoginServiceTest.signupSuccess » IllegalState Failed to load ApplicationContex...
  LoginServiceTest.signupUserExist » IllegalState Failed to load ApplicationCont...
  UserInformationServiceTest.searchSuccess » IllegalState Failed to load Applica...

Tests run: 71, Failures: 0, Errors: 8, Skipped: 2

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 02:07 min
[INFO] Finished at: 2017-05-26T21:35:43+00:00
[INFO] Final Memory: 35M/211M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [KoralChapnik] f051fea37a65277bcf3b6fdb40ec1a3f92725f43 - implemented server side #382

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Unit%20Tests/717/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
